### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**:

### CODE REVIEW

File: db/re/item_combo_db.txt [hunk 1]
```diff
@@ -2569,3 +2571,52 @@
 450142:4135,{.@r = getequiprefinerycnt(EQI_ARMOR); bonus bNoWalkDelay; bonus2 bResEff,Eff_Freeze,10000; if (BaseLevel >= 100) { bonus2 bAddEle,Ele_Dark,10*.@r; } else if (BaseLevel <= 99) { bonus2 bAddEle,Ele_Dark,3*.@r; } }
 300308:27331,{ bonus bMatkRate,3; }
 300310:27333,{ bonus2 bAddRace,RC_All,3; }
+5932:4161,{ bonus3 bAutoSpell,"PR_GLORIA",5,20; }
+5932:1737,{ .@r = getequiprefinerycnt(EQI_HAND_R); if (BaseClass == Job_Ranger) { bonus2 bSkillAtk,"HT_BLITZBEAT",5*.@r; } }
+5932:18129,{ .@r = getequiprefinerycnt(EQI_HAND_R); if (BaseClass == Job_Ranger) { bonus2 bSkillAtk,"HT_BLITZBEAT",5*.@r; } }
+5934:4206,{ if (readparam(bLUK)>=77) { bonus2 bAddMonsterDropItem,22839,30; } /*Unknow Rate*/ }
+15136:4191,{ .@r = getrefine(); if (.@r==5) { bonus bDefEle,Ele_Earth; } if (.@r==6) { bonus bDefEle,Ele_Water; } if (.@r==7) { bonus bDefEle,Ele_Wind; } if (.@r==8) { bonus bDefEle,Ele_Fire; } if (.@r==9) { bonus bDefEle,Ele_Dark; } if (.@r>=10) { bonus bDefEle,Ele_Holy; } }
```
Comment (Atemo) [inline db/re/item_combo_db.txt:2578]:
```suggestion
15136:4191,{ .@r = getequiprefinerycnt(EQI_ARMOR); if (.@r==5) { bonus bDefEle,Ele_Earth; } else if (.@r==6) { bonus bDefEle,Ele_Water; } else if (.@r==7) { bonus bDefEle,Ele_Wind; } else if (.@r==8) { bonus bDefEle,Ele_Fire; } else if (.@r==9) { bonus bDefEle,Ele_Dark; } else if (.@r>=10) { bonus bDefEle,Ele_Holy; } }
```

Apply to the others combo below as well

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -68177,6 +68222,62 @@ Body:
...
+    Locations:
+      Armor: true
+    ArmorLevel: 1
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      .@lvl = getskilllv("AL_HEAL");
+      bonus bMaxHPrate,.@lvl;
+      bonus bLongAtkDef,@lvl;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:68273]:
```suggestion
      bonus bLongAtkDef,.@lvl;
```

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -68177,6 +68222,62 @@ Body:
...
+      autobonus3 "{ bonus bAspd,2; }",1000,10000,"TF_BACKSLIDING","{ specialeffect2 EF_HASTEUP; }";
+      bonus bDex,3*(.@r/4);
+      bonus bMaxHPrate,3*(.@r/4);
+      bonus2 bIgnoreDefRaceRate,RC_DemiHuman,15;
+      bonus2 bIgnoreDefRaceRate,RC_Player_Human,15;
+      bonus2 bIgnoreDefRaceRate,RC_Brute,15;
+      bonus2 bIgnoreDefRaceRate,RC_Player_Doram,15;
+      bonus2 bSkillAtk,"SC_TRIANGLESHOT",2*(getskilllv("AC_DOUBLE"));
+      if (.@r>=12)
+         bonus2 bSkillCooldown,"GC_DARKCROW",-5000;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:68252]: For me the skillatk bonus is in the +12 refine, the sentence being after the +12 refine check (like you did for the next item)
```suggestion
      if (.@r>=12) {
         bonus2 bSkillCooldown,"GC_DARKCROW",-5000;
         bonus2 bSkillAtk,"SC_TRIANGLESHOT",2*(getskilllv("AC_DOUBLE"));
      }
```

File: db/re/item_combo_db.txt [hunk 4]
```diff
@@ -2569,3 +2571,52 @@
...
+15136:4371,{ .@r = getequiprefinerycnt(EQI_ARMOR); bonus2 bSkillAtk,"PA_SHIELDCHAIN",5*(.@r/2); bonus2 bSkillAtk,"CR_DEVOTION",5*(.@r/2); }
+15136:4382,{ .@r = getrefine(); if (.@r==5) { bonus bDefEle,Ele_Earth; } if (.@r==6) { bonus bDefEle,Ele_Water; } if (.@r==7) { bonus bDefEle,Ele_Wind; } if (.@r==8) { bonus bDefEle,Ele_Fire; } if (.@r==9) { bonus bDefEle,Ele_Dark; } if (.@r>=10) { bonus bDefEle,Ele_Holy; } }
+15363:4303,{ bonus bBaseAtk,30; bonus bFlee,20; }
+15363:4580,{ bonus2 bSubEle,Ele_Holy,20; bonus2 bSubEle,Ele_Ghost,20; }
+15365:4160,{ bonus2 bSubRace,RC_All,5; }
+15365:4376,{ bonus bBaseAtk,50; bonus2 bAddClass,Class_Boss,25; bonus2 bMagicAddClass,Class_Boss,25; }
+18992:4230,{ .@r = getequiprefinerycnt(EQI_HEAD_TOP); bonus bAspdRate,.@r; }
+18992:4252,{ .@r = getequiprefinerycnt(EQI_HEAD_TOP); bonus bHPrecovRate,4*.@r; bonus bSPrecovRate,4*.@r; }
+18992:4327,{ .@r = getequiprefinerycnt(EQI_HEAD_TOP); bonus bVariableCastrate,-2*.@r; bonus2 bSkillAtk,"MG_FIREWALL",5*.@r; }
+19311:4115,{ .@r = getrefine(); bonus2 bHPDrainRate,-30,10; if (.@r>=9) { bonus2 bHPDrainRate,-30,30; } else if (.@r>=7) { bonus2 bHPDrainRate,-30,15; } }
```
Comment (Atemo) [inline db/re/item_combo_db.txt:2593]:
```suggestion
19311:4115,{ .@r = getequiprefinerycnt(EQI_HEAD_TOP); bonus2 bHPDrainRate,-30,10; if (.@r>=9) { bonus2 bHPDrainRate,-30,30; } else if (.@r>=7) { bonus2 bHPDrainRate,-30,15; } }
```

File: db/re/item_db_equip.yml [hunk 5]
```diff
@@ -100707,6 +100852,34 @@ Body:
...
+    Defense: 7
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 99
+    Refineable: true
+    Script: |
+      bonus bMdef,10;
+      bonus bDelayrate,-2*(getrefine()/2);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:100882]: The description is saying per refine (JRO)
```suggestion
      bonus bDelayrate,-2*getrefine();
```

File: db/re/item_combo_db.txt [hunk 6]
```diff
@@ -2569,3 +2571,52 @@
...
+22070:4274,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bHPGainValue,30; bonus bSPGainValue,3; bonus bBaseAtk,10*(.@r/3); bonus bMatk,10*(.@r/3); }
+22070:4289,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bHPGainValue,30; bonus bSPGainValue,3; bonus bBaseAtk,10*(.@r/3); bonus bMatk,10*(.@r/3); }
+22070:4307,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bHPGainValue,30; bonus bSPGainValue,3; bonus bBaseAtk,10*(.@r/3); bonus bMatk,10*(.@r/3); }
+22070:4308,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bHPGainValue,30; bonus bSPGainValue,3; bonus bBaseAtk,10*(.@r/3); bonus bMatk,10*(.@r/3); }
+22070:4316,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bHPGainValue,30; bonus bSPGainValue,3; bonus bBaseAtk,10*(.@r/3); bonus bMatk,10*(.@r/3); }
+22070:4329,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bHPGainValue,30; bonus bSPGainValue,3; bonus bBaseAtk,10*(.@r/3); bonus bMatk,10*(.@r/3); }
+22071:2234,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bVariableCastrate,-10+(.@r/2); bonus bMatkRate,(.@r/2); }
+22071:2235,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bVariableCastrate,-10+(.@r/2); bonus2 bAddRace,RC_All,(.@r/2); }
```
Comment (Atemo) [inline db/re/item_combo_db.txt:2605]:
```suggestion
22071:2234,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bVariableCastrate,-10-(.@r/2); bonus bMatkRate,(.@r/2); }
22071:2235,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bVariableCastrate,-10-(.@r/2); bonus2 bAddRace,RC_All,(.@r/2); }
```

File: db/re/item_combo_db.txt [hunk 7]
```diff
@@ -2569,3 +2571,52 @@
...
+22071:4236,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bVariableCastrate,-10; bonus3 bAutoSpellWhenHit,"PR_KYRIE",10,-(30+70*(readparam(bInt)>=99)); bonus bAllStats,3*(.@r); /*TODO: Cancel auto spell bonus is not implemented in rAthena yet.*/}
+22071:5164,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bVariableCastrate,-10+(.@r/2); bonus bMatkRate,(.@r/2); }
+22071:5165,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bVariableCastrate,-10+(.@r/2); bonus2 bAddRace,RC_All,(.@r/2); }
```
Comment (Atemo) [inline db/re/item_combo_db.txt:2608]:
```suggestion
22071:5164,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bVariableCastrate,-10-(.@r/2); bonus bMatkRate,(.@r/2); }
22071:5165,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bVariableCastrate,-10-(.@r/2); bonus2 bAddRace,RC_All,(.@r/2); }
```

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_combo_db.txt (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_combo_db.txt]: Thanks

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

File: db/re/item_combo_db.txt (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_combo_db.txt]: Thanks

File: db/re/item_db_equip.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:100882]: Thanks

File: db/re/item_combo_db.txt (hunk 6 above)
Comment (idk-whoami) [reply db/re/item_combo_db.txt]: Thanks

File: db/re/item_combo_db.txt (hunk 7 above)
Comment (idk-whoami) [reply db/re/item_combo_db.txt]: Thanks

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:68273]: Thanks

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**